docs/source/manual/auth.rst

```diff
@@ -216,3 +216,61 @@ resource method.
 If you have a resource which is optionally protected (e.g., you want to display a logged-in user's
 name but not require login), you need to implement a custom filter which injects a security context
 containing the principal if it exists, without performing authentication.
+
+Testing Protected Resources
+===========================
+
+Add this dependency into your ``pom.xml`` file:
+
+.. code-block:: xml
+
+    <dependencies>
+      <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-testing</artifactId>
+        <version>${dropwizard.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+        <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
+        <version>2.19</version>
```

Comment:
Maybe just use a placeholder like `${jersey.version}` so that we don't have to update the docs for every Jersey update.